features/step_definitions/operators.rb

```diff
@@ -168,6 +168,7 @@
   interval_time = 5
   timeout = Integer(timeout) rescue 60
   interval_time = 20 if timeout > 100
+  interval_time = 5 if env.nodes.length == 1 # interval needs to be short because rotation is quick in SNO env
```

Comment:
Check line 168, `interval_time` defaults to 5

Reply:
@liangxia line 170 overrides it, the case has steps of timeout > 100 (180 seconds), this caused the failure, see the Description part:
```
#<RuntimeError: The authentication operator still didn't become {"Progressing"=>"True"} after 180 seconds>
```

Reply:
Perhaps that means you need a shorter timeout rather than overwrite it again

Reply:
We need it due to https://github.com/openshift/verification-tests/pull/2218/files#diff-8efd8ed73179d3b17bdfae8adad9cf49e0a0ad91eaf68e3b2a36766289d56616R41 .

Reply:
Could you help check again per previous comment?

Reply:
@liangxia @pruan-rht , could you help merge if no more objection given above reasons explained? Thanks!

Reply:
That's for the rollout, but here we are talking about operator in progressing.

It should work if you change 
`operator "authentication" becomes progressing within 180 seconds` to 
`operator "authentication" becomes progressing within 100 seconds`

Reply:
`operator "authentication" becomes progressing within 100 seconds` met failures due to https://github.com/openshift/verification-tests/pull/2218/files#diff-8efd8ed73179d3b17bdfae8adad9cf49e0a0ad91eaf68e3b2a36766289d56616R41 .

Reply:
So the workaround for the bug spreads here and there.  Any plan to cleanup those workarounds after the bug is fixed ?

Reply:
> here and there

Understood your concern, and frankly speaking, I hate workarounds too, just had to do.
About "here and there", I have to explain "here and there" are both needed. For "there" (i.e. let the timeout big enough), it is due to the bug [1958198](https://bugzilla.redhat.com/show_bug.cgi?id=1958198) there. For "here" (i.e. let `interval_time = 5 if env.nodes.length == 1`), it is due to SNO feature that transitions statuses quicker. Please don't worry about cleanup, when ON_QA we of course we'll remember to clean up. Thanks!